sdk/tables/azure-data-tables/migration_guide.md

```diff
@@ -0,0 +1,289 @@
+
+To improve the development experience across Azure services, a set of uniform [design guidelines](https://azure.github.io/azure-sdk/general_introduction.html) were created for all languages to drive a consistent experience with established API patterns for all services. A set of [Python guidelines](https://azure.github.io/azure-sdk/python_design.html) were also introduced to ensure that Python clients have a natural and idiomatic feel with respect to the Python ecosystem. The new `azure-data-tables` follows these guidelines.
+
+### Cross Service SDK improvements
+
+The modern `azure-data-tables` client library also provides the ability to share in some of the cross-service improvements made to the Azure development experience, such as
+<!-- # Not used in this library, should we still include?
+- Using the new `azure-identity` library to share a single authentication approach between clients -->
+- A unified logging and diagnostics pipeline offering a common view of the activities across each of the client libraries
+
+### Performance improvements
+
+Use this section to advertise the performance improvements in new package when compared to the old one. Skip this section if no perf improvements are found yet.
+
+### New features
+
+We have a variety of new features available in the new library:
+* Ability to submit a batch of operations with the `TableClient.submit_transaction` method
+
+## Important changes
+
+### Package names and namespaces
+
+The package name has been changed from `azure-cosmosdb-table` to `azure-data-tables`. This package can target either CosmosDB or Azure Storage Tables accounts.
+
+### Client hierarchy and constructors
+
+In the interest of simplicity, there are only two clients, `TableServiceClient` for account-level interactions and `TableClient` for table-level interactions. This is in contrast to the single `CosmosClient` used for interacting with the account level operations and the proxies used for interactions with specific resources (ie. `DatabaseProxy` and `ContainerProxy`).
+
+### Authenticating Clients
+
+In `azure-cosmosdb-table`:
+```python
+import os
+from azure.cosmosdb.table import TableService
+
+client = TableService(account_name=os.environ["ACCOUNT_URI"], account_key=os.environ["ACCOUNT_KEY"])
+```
+
+In `azure-data-tables`:
+```python
+import os
+from azure.core.credentials import AzureNamedKeyCredential
+from azure.data.tables import TableClient, TableServiceClient
+
+key = os.environ["TABLES_ACCOUNT_KEY"]
+name = os.environ["TABLES_ACCOUNT_NAME"]
+url = os.environ["TABLES_ACCOUNT_URL"]
+
+credential = AzureNamedKeyCredential(key=key, name=name)
+
+table_client = TableClient(account_url, "tablename", credential=credential)
+
+table_service_client = TableServiceClient(account_url, credential=credential)
+```
+
+
+### Table Level Scenarios
```

Comment:
in general, for scenarios below, answer the question:
Do Cosmos and Tables achieve the same thing? Then, why is it better to use Tables than Cosmos, esp from customer's standpoint?

Reply:
In general Cosmos and tables achieve the same thing, there are certain properties a user can set on Tables only (which we don't display here). It's mostly a customer choice.

Reply:
@swathipil - are you asking why I would use CosmosTables vs StorageTables? Or why I would use T1 SDK vs T2 SDK?

Reply:
^^ T1 vs. T2